edi_oca/models/edi_backend.py

```
@@ -579,6 +577,17 @@ def _exchange_receive(self, exchange_record):
             return component.receive()
         raise NotImplementedError()
 
+    def _process_received_content(self, exchange_record, content):
+        if not content:
```

we have 2 cases:
- content is None: -> because of FileNotFoundError/OSError, we don't do anything
- `len(content) == 0`: file is empty

---

We use `len()` with `False or None` will raise Error
```
>>> a = None
>>> len(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()
```
```
>>> len(False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'bool' has no len()
```
So I updated the PR

---

in which case do we have False?

---

For now, we don't have any case with `False` value for content, but in the future, maybe it will support some storage that wants to ignore the error and return `False` value

I think it will be great if we bypass both `None` and `False` in case the storage wants to ignore the error by returning one of them

---

I don't really care but usually we try to cover existing cases not future potential cases